platform/windows/os_windows.cpp

```diff
@@ -435,8 +435,6 @@ String OS_Windows::get_distribution_name() const {
 }
 
 String OS_Windows::get_version() const {
-	typedef LONG NTSTATUS;
-	typedef NTSTATUS(WINAPI * RtlGetVersionPtr)(PRTL_OSVERSIONINFOW);
 	RtlGetVersionPtr version_ptr = (RtlGetVersionPtr)GetProcAddress(GetModuleHandle("ntdll.dll"), "RtlGetVersion");
```

Comment:
I'd like someone to confirm this works across cases, it shadows both names but `RtlGetVersionPtr` is defined in display server under windows

Reply:
@bruvzg

Reply:
Should be fine, it's used in both DisplayServer and OS, and only one define is required since `os_windows.cpp` includes `display_server_windows.h` (which have it defined).